src/View/View.php

```diff
@@ -1260,7 +1262,9 @@ protected function addHelper(string $helper, array $config = []): void
     }
 
     /**
-     * Loads a helper. Delegates to the `HelperRegistry::load()` to load the helper
+     * Loads a helper. Delegates to the `HelperRegistry::load()` to load the helper.
+     *
+     * Deprecated: The use within `initialize()` method is deprecated, use `addHelper()` instead.
```

Comment:
This is fine in principle but technically there's no way for use to prevent it being used from `initialize()`. So perhaps the message should be changed and instead of saying it's deprecated just mention that it's recommended to use `addHelper()` instead of this method in `initialize()`.

Reply:
True. I thought soft a soft "Deprecated: " term is used here for this normally, since this is not technically enforced or enforceable. I can rephrase though.